bedrock/settings/static_media.py

```diff
@@ -428,6 +428,17 @@
         ),
         'output_filename': 'css/firefox_os-bundle.css',
     },
+    'firefox_os_new': {
```

Comment:
Could I suggest we rename this bundle to `firefox_os_2_5` or something a little more descriptive?